docs/your-first-script.md

```diff
@@ -2,115 +2,209 @@
 
 # Your first script
 
+Once you have Nextflow installed, you’re ready to run your first script. This guide details fundamental skills to run a basic Nextflow pipeline. It includes:
+
+- Running a pipeline
+- Modifying and resuming a pipeline
+- Configuring a pipeline parameter
+
+<h3>Prerequisites</h3>
+
+You will need the following to get started:
+
+- Nextflow 23.10.0 or later. See {ref}`install-page` for instructions to install or update your version of Nextflow.
```

Comment:
It's a bit awkward to mention this prerequisite when you already started by saying "once you have Nextflow installed...". Is there a way we could mention this once?

Maybe we don't need to give the minimum version? At this point the 23.10 is pretty arbitrary, if the only requirement is DSL2, that's been standard for so long that maybe we can assume that if the user goes through the Install page, they will have a sufficient version

Reply:
Fair point about the repetition. I'll revise the text.

I would like to introduce a "Prerequisites" section to all guides and tutorials so that it's clear what is required to complete the guide. It's not as important here as all we expect is Nextflow, but as we add more guides that require more details (e.g., credentials, docker), I'd like to have some consistency.

Reply:
Update, I've removed the "once you have nextflow installed" text but kept the Prerequisites section. If you think it's too thin I could add another point about a text editor, and then point at the VS code or Dev Env page.

Reply:
Sounds good to me. Let's merge!